packages/css/src/components/ListItem.css

```diff
@@ -0,0 +1,81 @@
+.ListItem {
+  padding: var(--space-l);
+  position: relative;
+  display: flex;
+  display: -webkit-flex;
+  flex-direction: row;
+  align-items: center;
+  min-width: 320px;
```

Comment:
Shouldn't this be `100%`?

Reply:
🤔 Haven't thought about it. Asking Juan...

Reply:
And he is `yes` :)
Changing!